### PR TITLE
Convert fabric-expanding-v1 to ES6

### PR DESCRIPTION
**Pull request by janua**
👏 2 to go 👏 

`this.params.id = 'fabric-expanding-' + (Math.random() * 10000 | 0).toString(16);`
has changed to
`this.params.id = 'fabric-expanding-' + (Math.floor(Math.random() * 10000).toString(16);`

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/creatives/fabric-expanding-v1.js [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+        this.initialExpandCounter = false;
+
+        this.closedHeight = 250;
+        this.openedHeight = 500;
+
+        bindAll(this, 'updateBgPosition', 'listener');
+    }
+
+    updateBgPosition() {
+        const that = this;
```
Comment (regiskuckaertz) [inline static/src/javascripts/projects/commercial/modules/creatives/fabric-expanding-v1.js]: you don't need `that`

**Comment by janua** [comment]
@regiskuckaertz Done 😎

**Comment by janua** [comment]
@regiskuckaertz Could you plus 1 if you are happy? 🙏

**Review by regiskuckaertz — APPROVED** [review]
rock'n'roll

**Pull request merged**